src/core/index.js

```diff
@@ -37,31 +37,3 @@ exports._test = {
 exports.sjcl = require('./utils').sjcl;
 exports.Wallet = require('ripple-wallet-generator')({sjcl: exports.sjcl});
 exports.types = require('./serializedtypes');
-
-// camelCase to under_scored API conversion
-function attachUnderscored(name) {
```

Comment:
Yeah, the idea is to unexpose core so that we can freely refactor/delete code without any concern about breaking things. There is no set timeframe for removal, but we're trying to keep things moving fast.